metadata.json

```diff
@@ -10,7 +10,7 @@
   "description": "This module provides type/providers for syslog using the Augeas configuration API library.",
   "dependencies": [
     {
-      "name": "herculesteam/augeasproviders_core",
+      "name": "puppet/augeasproviders_core",
       "version_requirement": ">=2.4.0 <3.0.0"
```

Comment:
do you also want to allow verion 3.x and 4.x?

Reply:
can do, wanted to originally do it in a followup PR.